tests/baselines/reference/docker/azure-sdk.log

```diff
@@ -63,7 +64,7 @@ Executing a maximum of ?simultaneous processes...
 "@azure/event-processor-host" is blocked by "@azure/eslint-plugin-azure-sdk".
 "testhub" is blocked by "@azure/eslint-plugin-azure-sdk".
 "@azure/logger" is blocked by "@azure/eslint-plugin-azure-sdk".
-==[ BLOCKED: 57 projects ]=====================================================
+==[ BLOCKED: 58 projects ]=====================================================
```

Comment:
looks like azure-sdk is still blocked by not installing (or a bad install?) `@typescript-eslint/scope-manager`

Reply:
Is this our issue or an issue in `azure-sdk`? I didn't do anything for `azure-sdk` in #42431, so as far as I can tell this is just the latest build output from `master`.